package.json

```diff
@@ -10,42 +10,39 @@
     "@geops/react-ui": "1.1.1-beta.3",
     "@material-ui/core": "4.11.0",
     "@material-ui/lab": "4.0.0-alpha.56",
-    "@turf/intersect": "6.1.3",
-    "@webcomponents/custom-elements": "1.4.1",
+    "@turf/intersect": "6.3.0",
+    "@webcomponents/custom-elements": "1.4.3",
     "@webcomponents/webcomponents-platform": "1.0.1",
-    "@webcomponents/webcomponentsjs": "2.4.3",
-    "abortcontroller-polyfill": "1.5.0",
-    "color": "3.1.2",
+    "@webcomponents/webcomponentsjs": "2.5.0",
+    "abortcontroller-polyfill": "1.7.1",
     "deepmerge": "4.2.2",
     "i18next": "19.3.4",
     "i18next-browser-languagedetector": "4.0.2",
     "i18next-xhr-backend": "3.2.2",
-    "jsts": "2.1.0",
     "mapbox-gl": "1.11.1",
     "mobility-toolbox-js": "1.2.12",
     "node-sass": "4.13.1",
     "ol": "6.3.1",
-    "proj4": "2.6.1",
+    "proj4": "2.7.0",
     "prop-types": "15.7.2",
     "proxy-polyfill": "0.3.1",
-    "query-string": "6.13.1",
-    "react": "16.13.1",
+    "query-string": "6.14.1",
+    "react": "16.14.0",
```

Comment:
react 17 is out

Reply:
Start the work to update to react v17 but it seems not that easy:
https://github.com/geops/trafimage-maps/pull/894

Some test libs are not yet up to date, for example 'enzyme-adapter-react-17' is not officialy published:
https://github.com/enzymejs/enzyme/issues/2429

I would leave that open until react update is doable easily.

Reply:
I agree